std/stdio.d

```diff
@@ -4572,16 +4571,17 @@ private struct ChunksImpl
 
 /**
 Writes an array or range to a file.
-Shorthand for $(D data.copy(File(fileName, "wb").lockingBinaryWriter)).
+Shorthand for `put(data, File(fileName, "wb").lockingBinaryWriter)`.
 Similar to $(REF write, std,file), strings are written as-is,
 rather than encoded according to the $(D File)'s $(HTTP
 en.cppreference.com/w/c/io#Narrow_and_wide_orientation,
 orientation).
 */
 void toFile(T)(T data, string fileName)
-if (is(typeof(copy(data, stdout.lockingBinaryWriter))))
+if (is(typeof({ auto w = stdout.lockingBinaryWriter; put(w, data); })))
 {
-    copy(data, File(fileName, "wb").lockingBinaryWriter);
+    auto w = File(fileName, "wb").lockingBinaryWriter;
+    put(w, data);
```

Comment:
I'd prefer reverting this, but I'm fine with the new version as well.

Reply:
I left this one to remove the global std.algorithm.mutation import.